tests/foreman/cli/test_host.py

```diff
@@ -79,3 +82,205 @@ def test_create_libvirt_without_mac(self):
             u'puppet-proxy-id': self.puppet_proxy['id'],
             u'root-pass': host.root_pass,
         })
+
+
+class HostParameterTests(CLITestCase):
+    """Tests targeting host parameters"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create host to tests parameters for"""
+        super(HostParameterTests, cls).setUpClass()
+        cls.puppet_proxy = Proxy.list()[0]
+        cls.host = entities.Host()
+        cls.host.create_missing()
+        cls.host = Host.create({
+            u'architecture-id': cls.host.architecture.id,
+            u'domain-id': cls.host.domain.id,
+            u'environment-id': cls.host.environment.id,
+            u'location-id': cls.host.location.id,  # pylint:disable=no-member
```

Comment:
Take a look at `nailgun.entities.Host.create_missing()`. All the others objects like `domain`, `environment` etc. are initialized there, but not `location` or `organization`. `create_missing()` is using existing location and organization; and as both are required fields, nailgun will create them if they were not provided.
And code analyzers are complaining as no member `location` or `organization` was directly initialized in `entities.Host` class.